src/tracker/tracker.jl

```diff
@@ -0,0 +1,63 @@
+#---------------------
+# TRACKER COMPONENTS
+#---------------------
+
+include("../core.jl")
+include("tracker_sampler.jl")
+# include("tracker_state_estimator.jl")
+# include("tracker_model.jl")
+# include("tracker_features.jl")
+
+abstract type AbstractTracker end
+mutable struct TrackerBoosting{I <: Integer, F <: AbstractFloat, B <: BoxROI} <: AbstractTracker
```

Comment:
Shouldn't the structs for the individual trackers be placed in their own separate files i.e. boosting_tracker.jl in this case?

Reply:
My intention of having them in a `tracker.jl` was to have a common umbrella for all the tracker definitions and have implementation details in seperate files i.e. `boosting_tracker.jl` in this case.